Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -474,11 +555,23 @@ async def main(argv):
         logger.critical("The EOL data in %s is outdated and we cannot reliably run this script.", EOLDATA_FILE)
         return 1
 
+    kubeconfig_path = config.kubeconfig
+
     connector = aiohttp.TCPConnector(limit=5)
     async with aiohttp.ClientSession(connector=connector) as session:
         cve_affected_ranges = await collect_cve_versions(session)
     releases_data = fetch_k8s_releases_data()
 
+    try:
+        logger.info(f"Checking cluster specified by {kubeconfig_path}")
```

Comment:
This needs some more explanation, because almost the same message will be displayed in line 577 (only better, because there, it contains the context as well).

Reply:
My point was that it looks like a duplicate of the other line, and it appears to me that the script now tries to achieve the same objective with two different means, one after the other.